docs/product_catalog_management/manage_products.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Add and configure products in your catalog, set up their attributes and define availability and stock.
+---
+
+# Manage products
+
+In [[= product_name =]] you store aggregate product information in Products.
```

Comment:
```suggestion
In [[= product_name =]] you store aggregate product information in PIM.
```

Reply:
Done! The UI however still uses "Product catalog" as a caption for the PIM icon. Whenever I refer to a menu option, I use the icon name. I'll fix that when the UI changes.